src/Workspaces/Core/Portable/CodeActions/CodeActionProviderPriority.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CodeActions
+{
+    internal enum CodeActionProviderPriority
```

Comment:
I admit the naming here is a bit confusing compared to CodeActionPriority; this is only really used to filter when we're computing, right? i.e. this isn't returned from CodeActionProvider anywhere. Maybe call it "CodeActionRequestedPriority" or something?

Reply:
I like that. Thanks!